front/lib/api/assistant/configuration.ts

```diff
@@ -111,6 +106,18 @@ const sortStrategies: Record<SortStrategyType, SortStrategy> = {
 export async function getAgentConfiguration(
   auth: Authenticator,
   agentId: string
+): Promise<LightAgentConfigurationType | null> {
```

Comment:
This change is quite tricky. Can we carve it off in a different PR for review and rebase on it :pray: